cylc/flow/scheduler_cli.py

```diff
@@ -612,8 +609,12 @@ def _distribute(host, workflow_id_raw, workflow_id, color):
         # Prevent recursive host selection
         cmd.append("--host=localhost")
 
+        # Ensure interactive upgrade carries over:
+        if options.upgrade and '--upgrade' not in cmd:
+            cmd.append('--upgrade')
```

Comment:
Needs duplicating for `--downgrade`?

Reply:
There isn't an interactive downgrade though

Reply:
Gotcha.